oaknational/Oak-Web-Application
# ENG-482 fix(lesson scroll behaviour): check previous section not completely i…

## Description

- check previous section not completely in view before setting the next section to be current

## How to test

1. Go to https://deploy-preview-1988--oak-web-application.netlify.thenational.academy/teachers/programmes/biology-secondary-ks4-l/units/cells-biology-3p8njnj/lessons/prokaryotic-and-eukaryotic-cells-b9qqye
2. Click on **Lesson details**
3. You should see lesson details is the current and higlighted section

## Screenshots

![image](https://github.com/oaknational/Oak-Web-Application/assets/12934669/97705f45-7b09-44a6-97b8-0ec89f367ac3)
